www/includes/easyparliament/templates/html/mp/recent.php

```diff
@@ -61,15 +61,19 @@
             </div>
 
             <div class="sidebar__unit in-page-nav">
-                <ul>
-                  <?php foreach($sidebar_links as $date) { ?>
-                    <li>
-                        <a href="#<?= strftime('%Y-%m-%d', strtotime($date)) ?>">
-                            <?= strftime('%e %b %Y', strtotime($date)) ?>
-                        </a>
-                    </li>
-                  <?php } ?>
-                </ul>
+                <div>
+                    <h3 class="browse-content">Browse content</h3>
+                    <ul>
+                        <?php foreach($sidebar_links as $date) { ?>
+                          <li>
+                              <a href="#<?= strftime('%Y-%m-%d', strtotime($date)) ?>">
+                                  <?= strftime('%e %b %Y', strtotime($date)) ?>
+                              </a>
+                          </li>
+                        <?php } ?>
+                      </ul>
+                      <?php include '_featured_content.php'; ?>
```

Comment:
presumably donation not in this one on purpose?

Reply:
As I remember, the long side bar pushed it out of visibility anyway. so I didn't include it for neatness.

Reply:
Don't follow this, sorry, as /votes sidebar seems far longer than /recent?

Reply:
Ah, I had it out of the voting summaries on purpose, recent is an oversight.

Reply:
Yeah, this is an oversight - I was leaving it out of a different page (have just put it everywhere for consistency now).